### ISSUE TITLE
How to polyfill picture in gatsbyjs

### ISSUE THREAD

**Issue by TylerBarnes** [gatsbyjs/gatsby#11558]
Hey there,
I've been trying to polyfill the picture tag using [picturefill](https://scottjehl.github.io/picturefill/).
I've tried all 3 of the following with no luck; I still get "the picture tag is unrecognized in this browser" in IE11.

```jsx
exports.onClientEntry = async () => {
  await import("picturefill/dist/plugins/mutation/pf.mutation");
  const picturefill = await import("picturefill");
  picturefill.default();
};
```

```jsx
require("picturefill/dist/plugins/mutation/pf.mutation");
const picturefill = require("picturefill");

exports.onClientEntry = () => picturefill();
```

```jsx
const picturefill = require("picturefill");

exports.onClientEntry = () => picturefill();

exports.onRouteUpdate = () => picturefill();
```

https://pfmutations-test.netlify.com/
https://github.com/TylerBarnes/pfmutations-test-gatsby

Is there a standard way of polyfilling picture in gatsby? These don't seem to work.

**Comment by KyleAMathews** [comment]
Polyfilling picture isn't strictly necessary (as you perhaps know) as older browsers will fall back on img.

**Comment by DanielRuf** [comment]
Well, picturefill also introduces support for responsive images in IE11. Most clients and websites do not want the img fallback - so we use picturefill.

Generally picturefill and the IE code for the picture element should be loaded before all other scripts afaik.

**Comment by DanielRuf** [comment]
It seems it worked for others in previous releases, see https://github.com/gatsbyjs/gatsby/issues/7148

**Comment by TylerBarnes** [comment]
@KyleAMathews that's true but it would be nice to be able to set this up once and keep using it in future gatsby projects. Improvement is improvement :) even for IE users. 
@DanielRuf I did see that before I opened this but yes, it doesn't seem to work in the current version. I guess that issue alone doesn't show us if it was actually working for @rtm619 though.
Maybe I'll try adding it to html.js tomorrow if it should load before everything else.
